### DISCUSSION TITLE
gesp-1-bcqm3016/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#337] [General]
# gesp-1-bcqm3016/

变量值读取练习，开始接触从控制台读取变量的值。（之前的BCQM3019]其实已经用过，但是如果按顺序的话，这是第一次接触）

https://www.coderli.com/gesp-1-bcqm3016/

**Comment by lihongzheshuai** [comment]
原题输入未说明是整数，这里我改成整数。否则有歧义。

**Comment by Niuger** [comment]
2024.9.28打卡
